src/main/scala/io/iohk/ethereum/healthcheck/Healthcheck.scala

```diff
@@ -18,20 +19,20 @@ import scala.util.{Failure, Success}
   */
 case class Healthcheck[Error, Result](
     description: String,
-    f: () => Future[Either[Error, Result]],
+    f: () => Task[Either[Error, Result]],
```

Comment:
Minor - it doesn't have to be a function now

Reply:
Good spot. Changed `() => Task` into `Task`